content/faq/_index.md

```diff
@@ -4,8 +4,14 @@ title="FAQ"
 
 ## Frequently Asked Questions
 
-#### What languages and frameworks are supported?
-We currently support Java, Node.js, Golang, .NET Core, PHP, and NGINX apps 
+#### What do Paketo Buildpacks build and where can I run software packaged with Paketo Buildpacks?
+Paketo Buildpacks build container images that run on Kubernetes and any other Cloud Native platform.
+
+#### How are containers built with Paketo different from Docker containers?
+Unlike Docker containers, containers built from Paketo Buildpacks are OCI compliant.
```

Comment:
@ryanmoran does this make sense, or any clarifications you would add here?

Reply:
I would use "image" instead of "container" here. To me, an image is a filesystem snapshot that can be run against a kernel, whereas a container is a running instance of that filesystem, kernel, and a specific process. The buildpacks produce images as their outputs.

Reply:
Thanks! updated.